src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/TestFixtures/SQLSetupStrategy.cs

```diff
@@ -272,17 +272,22 @@ public PlatformSpecificTestContext()
             {
                 certStoreFixture = new SQLSetupStrategyCertStoreProvider();
             }
-#if !NET46
             else
             {
                 akvFixture = new SQLSetupStrategyAzureKeyVault();
             }
-#endif
         }
 
         public void Dispose()
         {
-            Fixture.Dispose();
+            try
```

Comment:
Why don't we use the initial Fixture which will either be `certStoreFixture` or `akvFixture`?

Reply:
.NET 5 compiler raises error that these fixtures are not disposed.

Reply:
If that is the case, maybe simply use this?
```
akvFixture?.Dispose();
certStoreFixture?.Dispose();
```